baybe/constraints/conditions.py

```diff
@@ -13,6 +14,8 @@
 
 from baybe.serialization import SerialMixin
 
+PolarsType = Union[float, int, np.ndarray]
+
```

Comment:
I added this type to fix mypy type mismatch errors.

Reply:
See my answer [here](https://github.com/emdgroup/baybe/pull/252/files#r1622982252)

Reply:
Fixed.